rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -8425,6 +8425,182 @@ private static Stream<Arguments> getEstimatedFeesForNextPegOutEventArgsProvider_
         );
     }
 
+    private static Stream<Arguments> getEstimatedFeesForNextPegOutEventArgsProvider_post_RSKIP305(BridgeConstants bridgeConstants) {
+
+        ActivationConfig.ForBlock tbd800Activations = ActivationConfigsForTest.tbd800().forBlock(0L);
+
+        FederationConstants federationConstants = bridgeConstants.getFederationConstants();
+
+        List<FederationMember> members = FederationMember.getFederationMembersFromKeys(
+            PegTestUtils.createRandomBtcECKeys(7)
+        );
+
+        FederationArgs p2shFedArgs = new FederationArgs(members,
+            Instant.now(),
+            1L,
+            bridgeConstants.getBtcParams()
+        );
+
+        ErpFederation p2shFed =
+            FederationFactory.buildP2shErpFederation(p2shFedArgs, federationConstants.getErpFedPubKeysList(), federationConstants.getErpFedActivationDelay());
+
+        ErpFederation p2shP2wshFed =
+            FederationFactory.buildP2shP2wshErpFederation(p2shFedArgs, federationConstants.getErpFedPubKeysList(), federationConstants.getErpFedActivationDelay());
+
+        ReleaseRequestQueue.Entry pegoutRequest1 = new ReleaseRequestQueue.Entry(new BtcECKey().toAddress(bridgeConstants.getBtcParams()), Coin.valueOf(1, 0));
```

Comment:
This too should be deterministic. You can use BitcoinTestUtils to create keys from a seed

Reply:
Updated.